master/main.go

```diff
@@ -77,6 +77,7 @@ func main() {
 	router.HandleFunc("/addworker", AddWorkerRequest).Methods("GET")
 	router.HandleFunc("/processgraph", ProcessGraph).Methods("POST")
 	router.HandleFunc("/worker/register", registerWorker).Methods("POST")
+	router.HandleFunc("/worker/done", workerDoneProcessing).Methods("POST")
```

Comment:
Ik denk dat een get request hier netter is.

Reply:
Ik ook